app/charts/map/map.tsx

```diff
@@ -403,6 +403,8 @@ export const MapComponent = () => {
           initialViewState={defaultViewState}
           mapLib={maplibregl}
           mapStyle={mapStyle}
+          // Important so we can take a screenshot of the map
```

Comment:
Do we still need this comment?

Reply:
I would say yes, as `preserveDrawingBuffer` is `false` by default and can decrease the performance of the map, the comment should make sure that we don't remove it and unknowingly break screenshotting of the maps :)